utils/featuresConfigUtils.ts

```diff
@@ -0,0 +1,60 @@
+import Stripe from "stripe";
+
+export const defaultFeaturesConfig = {
+  payments: {
+    refund_management: true,
+    dispute_management: true,
+    capture_payments: true,
+  },
+  payment_details: {
+    refund_management: true,
+    dispute_management: true,
+    capture_payments: true,
+  },
+};
+
+export type FeaturesConfig = typeof defaultFeaturesConfig;
+
+export const featuresCache: {
```

Comment:
I see we are using this features cache module scoped variable to store the information - technically we should be able to do this in state only, however I think this can be simplified if we just do this via local storage, see my other comment suggestion!